lib/AST/ASTContext.cpp

```diff
@@ -3951,8 +3951,17 @@ bool ASTContext::isStandardLibraryTypeBridgedInFoundation(
           nominal == getStringDecl() ||
           nominal == getErrorDecl() ||
           nominal == getAnyHashableDecl() ||
-          // Weird one-off case where CGFloat is bridged to NSNumber.
-          nominal->getName() == Id_CGFloat);
+          // Foundation is a dependency of CoreGraphics, but
```

Comment:
Sorry, I meant "Foundation depends on CoreGraphics". Treating all imported types as possibly bridged might be prudent for ABI compatibility, if we want to reserve the right to continue adding new bridging relationships after ABI stability is declared.